src/typography.css

```diff
@@ -150,64 +150,71 @@ textarea {
  * <div class='txt-xs'>Malesuada Pharetra Ridiculus</div>
  */
 .txt-headline {
-  font-size: 32px;
-  line-height: 42px;
+  /* line height ratio: 1.2 */
+  font-size: 45px;
+  line-height: 54px;
 }
 
 .txt-subhead {
-  font-size: 28px;
-  line-height: 36px;
+  /* line height ratio: 1.2 */
+  font-size: 35px;
+  line-height: 42px;
```

Comment:
Curious: what's the reason you use a pixel value here instead of `line-height: 1.2` directly?